src/promptflow-evals/tests/evals/unittests/test_content_safety_rai_script.py

```diff
@@ -7,12 +7,8 @@
 import pytest
 from azure.identity import DefaultAzureCredential
 
-from promptflow.evals.evaluators._content_safety.common.constants import (
-    EvaluationMetrics,
-    HarmSeverityLevel,
-    RAIService,
-)
-from promptflow.evals.evaluators._content_safety.common.evaluate_with_rai_service import (
+from promptflow.evals.common.constants import EvaluationMetrics, HarmSeverityLevel, RAIService
```

Comment:
I think it's better to make it as an internal namespace: `promptflow.evals._common.constants`

Reply:
done